### PR TITLE
radicle org post

### PR DESCRIPTION
**Pull request by juliendonck**
Signed-off-by: Julien Donck <hello@juliendonck.com>

- [x] Replace Markdown links with HTML links
- [x] Review for spelling errors
- [x] Make sure links are correct
- [x] Add new gif (maybe one of these @brandonhaslegs)
https://media.giphy.com/media/3o7TPzzQse5ROPpnYQ/giphy.gif
https://media.giphy.com/media/U4FkC2VqpeNRHjTDQ5/giphy.gif
https://media.giphy.com/media/3xz2BvbzkugseSJJxm/giphy.gif

### CODE REVIEW

**Review by lftherios — APPROVED** [review]
:fireworks:

File: blog/index.html.mustache [hunk 1]
```diff
@@ -25,6 +25,18 @@
...
     <div class="overview">
+      <div class="post-item orgs">
+        <a href="/blog/radicle-orgs.html">
+          <h2 class="headline druk">Introducing Radicle Orgs</h2>
+        <video autoplay playsinline loop muted alt="Kid at a computer giving a thumbs up whent he Radicle token appears with fireworks around it.">
+          <source type="video/mp4" src="/img/seeders-post/seeders.mp4">
+        </video>
+        </a>
+        <div class="post-item-content">
+          <p class="meta">by <a href="https://twitter.com/lftherios" target="_blank">@abbey_titcomb</a> • <span>July 15th, 2021</span></p>
```
Comment (juliendonck) [inline blog/index.html.mustache:36]: The link is to `href="https://twitter.com/lftherios"` but the text says a`@abbey_titcomb`

File: blog/radicle-orgs.html.mustache [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        <p>
+          Widely used by Ethereum projects for treasury management, Gnosis Safe currently stewards [>$40billion USD][https://duneanalytics.com/tschubotz/gnosis-safe_2]. Although Gnosis Safe’s primary use is as an out-of-the-box asset management and coordination platform, its open-source architecture can also be modified by projects seeking to adapt its tooling for their own needs. 
+        </p>
+        <p>
+          “We view Radicle Org’s deployment on Gnosis Safe as an exciting open-source use of our tooling for aligned purposes",” says Gnosis CTO Stefan George. “Gnosis Safe is already the platform of choice for DAOs conducting decentralized treasury and membership management, and our integration with Radicle expands that application on a deeper level that we believe will provide greater service to the community.”
+        </p>
+        <p>
+          Building Radicle Orgs on Gnosis Safe’s composable architecture gives development teams the ability to manage and control membership, assets, and access rights to code repositories and other projects according to their needs.
+        </p>
+        <h2>On-chain, in-browser
```
Comment (juliendonck) [inline blog/radicle-orgs.html.mustache:75]:
```suggestion
        <h2>On-chain, in-browser</h2>
```

File: blog/radicle-orgs.html.mustache [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        <p>
+          Alongside the launch of Radicle Orgs comes the release of the Radicle web client, [app.radicle.network][https://app.radicle.network/]. Here, you can register ENS names for your orgs and customize their profiles profile. Our web client brings Radicle to the browser, making it easier to share your Radicle code with others.
```
Comment (juliendonck) [inline blog/radicle-orgs.html.mustache:77]:
```suggestion
          Alongside the launch of Radicle Orgs comes the release of the Radicle web client, [app.radicle.network][https://app.radicle.network/]. Here, you can register ENS names for your orgs and customize their profiles. Our web client brings Radicle to the browser, making it easier to share your Radicle code with others.
```

File: blog/radicle-orgs.html.mustache [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+        </p>
+        <p>
+          Alongside the launch of Radicle Orgs comes the release of the Radicle web client, app.radicle.network. Here, you can register ENS names for your orgs and customize their profiles profile. Our web client brings Radicle to the browser, making it easier to share your Radicle code with others.
```
Comment (juliendonck) [inline blog/radicle-orgs.html.mustache:80]: Duplicate paragraph

File: blog/radicle-orgs.html.mustache [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+        </p>
+        <p>
+          [On-chain anchors][https://docs.radicle.xyz/docs/connecting-to-ethereum/anchoring-projects.md] allow teams to maintain an auditable and transparent history of repositories on Ethereum. DAOs can now truly own their code while experimenting with decentralized admin control and collective governance of codebases. Interactions with orgs can also be used as a basis for distributing developer rewards, releasing software, or making important upgrades to codebases.
+        </p>
+        <p>
+          By framing DAOs and multi-sigs in the context of code collaboration, Radicle brings decentralized organizations to the mainstream developer, allowing anyone to decentralize control over their codebase in a trust-minimized way.
+        </p>
+        <h2>Radicle x Gnosis Safe</h2>
+        <p>
+          There are single-signer orgs (owned by one Ethereum address) or multi-signature orgs (owned by multiple Ethereum addresses). Multi-signature orgs are deployed with a [Gnosis Safe][https://gnosis-safe.io/]. This means Radicle users can deploy Safes to govern their org via the Radicle desktop or web client.
```
Comment (brandonhaslegs) [inline blog/radicle-orgs.html.mustache:58]:
```suggestion
          There are single-signer orgs (owned by one Ethereum address) or multi-signature orgs (owned by multiple Ethereum addresses). Multi-signature orgs are deployed with a [Gnosis Safe][https://gnosis-safe.io/]. This means Radicle users can deploy Safes to govern their org via Radicle Upstream or the Radicle web client.
```

Comment (brandonhaslegs) [reply blog/radicle-orgs.html.mustache:58]: So we're calling it the Radicle web client now?

File: blog/radicle-orgs.html.mustache [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+        </p>
+        <p>
+          “Our Gnosis Safe integration with Radicle Orgs gives DAOs access to a new superpower where membership, admin control, governance and canonical project state is truly decentralized,” explains Julien Donck who leads the team developing [Radicle Upstream](https://docs.radicle.xyz/docs/getting-started), the Radicle desktop client. “As a go-to primitive for DAOs, integrating with Gnosis Safe first was a no-brainer for us.”
```
Comment (brandonhaslegs) [inline blog/radicle-orgs.html.mustache:61]:
```suggestion
          “Our Gnosis Safe integration with Radicle Orgs gives DAOs access to a new superpower where membership, admin control, governance and canonical project state are truly decentralized,” explains Julien Donck who leads the team developing [Radicle Upstream](https://docs.radicle.xyz/docs/getting-started), the Radicle desktop client. “As a go-to primitive for DAOs, integrating with Gnosis Safe first was a no-brainer for us.”
```

File: blog/radicle-orgs.html.mustache (hunk 2 above)
Comment (brandonhaslegs) [inline blog/radicle-orgs.html.mustache:75]:
```suggestion
        <h2>On-chain, in-browser</h2>
```

File: blog/radicle-orgs.html.mustache [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+        </p>
+        <p>
+          [Gnosis Safe][https://gnosis-safe.io/] is a Web3 operating system for DAOs, teams, and individuals to manage digital assets. Its modular infrastructure enables users and developers to customize access control parameters and multi-sig permissions. Gnosis Safe’s functionality extends through the SafeSnap DAO module for on-chain execution of off-chain votes, and the Safe App ecosystem of dapps accessible directly from Safe’s secure interface.
+        </p>
+        <p>
+          Widely used by Ethereum projects for treasury management, Gnosis Safe currently stewards [>$40billion USD][https://duneanalytics.com/tschubotz/gnosis-safe_2]. Although Gnosis Safe’s primary use is as an out-of-the-box asset management and coordination platform, its open-source architecture can also be modified by projects seeking to adapt its tooling for their own needs.
```
Comment (brandonhaslegs) [inline blog/radicle-orgs.html.mustache:67]:
```suggestion
          Widely used by Ethereum projects for treasury management, Gnosis Safe currently stewards [more than 40 billion USD][https://duneanalytics.com/tschubotz/gnosis-safe_2]. Although Gnosis Safe’s primary use is as an out-of-the-box asset management and coordination platform, its open-source architecture can also be modified by projects seeking to adapt its tooling for their own needs. 
```

File: blog/radicle-orgs.html.mustache [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+        </p>
+        <p>
+          Widely used by Ethereum projects for treasury management, Gnosis Safe currently stewards [>$40billion USD][https://duneanalytics.com/tschubotz/gnosis-safe_2]. Although Gnosis Safe’s primary use is as an out-of-the-box asset management and coordination platform, its open-source architecture can also be modified by projects seeking to adapt its tooling for their own needs. 
+        </p>
+        <p>
+          “We view Radicle Org’s deployment on Gnosis Safe as an exciting open-source use of our tooling for aligned purposes",” says Gnosis CTO Stefan George. “Gnosis Safe is already the platform of choice for DAOs conducting decentralized treasury and membership management, and our integration with Radicle expands that application on a deeper level that we believe will provide greater service to the community.”
```
Comment (brandonhaslegs) [inline blog/radicle-orgs.html.mustache:70]:
```suggestion
          “We view Radicle Org’s deployment on Gnosis Safe as an exciting open-source use of our tooling for aligned purposes,” says Gnosis CTO Stefan George. “Gnosis Safe is already the platform of choice for DAOs conducting decentralized treasury and membership management, and our integration with Radicle expands that application on a deeper level that we believe will provide greater service to the community.”
```

File: blog/radicle-orgs.html.mustache (hunk 3 above)
Comment (brandonhaslegs) [inline blog/radicle-orgs.html.mustache:77]: > and customize their profiles profile.

not sure what this means

File: blog/radicle-orgs.html.mustache (hunk 4 above)
Comment (brandonhaslegs) [inline blog/radicle-orgs.html.mustache:80]:
```suggestion
```

File: blog/radicle-orgs.html.mustache [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+        <h2>On-chain, in-browser
+        <p>
+          Alongside the launch of Radicle Orgs comes the release of the Radicle web client, [app.radicle.network][https://app.radicle.network/]. Here, you can register ENS names for your orgs and customize their profiles profile. Our web client brings Radicle to the browser, making it easier to share your Radicle code with others.
+        </p>
+        <p>
+          Alongside the launch of Radicle Orgs comes the release of the Radicle web client, app.radicle.network. Here, you can register ENS names for your orgs and customize their profiles profile. Our web client brings Radicle to the browser, making it easier to share your Radicle code with others.
+        </p>
+        <hr>
+        <p>
+          To get started with Radicle Orgs, download [Radicle Upstream](https://radicle.xyz/downloads.html) and follow our docs!
```
Comment (brandonhaslegs) [inline blog/radicle-orgs.html.mustache:84]:
```suggestion
          To get started with Radicle Orgs, download [Radicle Upstream](https://radicle.xyz/downloads.html) and follow our [docs](docs.radicle.xyz)!
```

Comment (brandonhaslegs) [reply blog/radicle-orgs.html.mustache:84]: update to the right URL to link directly to the Org docs

**Review by brandonhaslegs — CHANGES_REQUESTED** [review]

File: blog/radicle-orgs.html.mustache [hunk 10]
```diff
@@ -80,7 +80,7 @@
         </p>
         <hr>
         <p>
-          To get started with Radicle Orgs, download [Radicle Upstream](https://radicle.xyz/downloads.html) and follow our docs!
+          To get started with Radicle Orgs, download [Radicle Upstream](https://radicle.xyz/downloads.html) and follow our [docs](docs.radicle.xyz)!
```
Comment (brandonhaslegs) [inline blog/radicle-orgs.html.mustache:83]: Can you make this link a more specific link to the Orgs docs?

**Review by brandonhaslegs — CHANGES_REQUESTED** [review]

File: blog/radicle-orgs.html.mustache [hunk 11]
```diff
@@ -77,7 +77,6 @@
           Alongside the launch of Radicle Orgs comes the release of the Radicle web client, [app.radicle.network][https://app.radicle.network/]. Here, you can register ENS names for your orgs and customize their profiles. Our web client brings Radicle to the browser, making it easier to share your Radicle code with others.
         </p>
         <p>
```
Comment (brandonhaslegs) [inline blog/radicle-orgs.html.mustache:79]: This paragraph tag needs to be removed.

**Review by brandonhaslegs — CHANGES_REQUESTED** [review]

**Review by abbey-titcomb — APPROVED** [review]

**Review by brandonhaslegs — APPROVED** [review]

**Pull request merged**